### DISCUSSION TITLE
open modal on massAction like bulk delete

### DISCUSSION THREAD

**Discussion by farshadff** [thedevdojo/voyager#5253] [Q&A]
hi guys ,
i have created a `massAction` the way it says in documentation but there is one problem and that's like bulk delete action i want to open a modal on clicking the mass action and show a modal before the action is triggered .  the reason i  or some other need this is because  you may want to customize the action some how like my case in below :
1- i have a group model
2-i have a user model
now in users list i want to select 10 users for example and press the `add to group` bottun a modal opens that in a dropdown shows the groups , so i select one groupd from dropdown and add the users to that group . 
i think this case may occur a lot in admin panels . so i would be thankfull if you guid me if there is any way to do this and add modal to mass action .
Note : i have asked this question on slack channel but seems like no one knows how its possible .
thanks

**Comment by emptynick** [comment]
Definitely not an issue -> moved to discussions.
